[Merged by Bors] - feat(Probability/Cdf): cumulative distribution function of a probability measure

RemyDegenne opened the pull request:
---
<!-- The text above the `---` will become the commit message when your
PR is merged. Please leave a blank newline before the `---`, otherwise
GitHub will format the text above it as a title.

To indicate co-authors, include lines at the bottom of the commit message
(that is, before the `---`) using the following format:

Co-authored-by: Author Name <author@email.com>

Any other comments you want to keep out of the PR commit should go
below the `---`, and placed outside this HTML comment, or else they
will be invisible to reviewers.

If this PR depends on other PRs, please list them below this comment,
using the following format:
- [ ] depends on: #abc [optional extra text]
- [ ] depends on: #xyz [optional extra text]
-->

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)

sgouezel commented on Mathlib/Probability/Cdf.lean:
```diff
@@ -0,0 +1,88 @@
+
+namespace ProbabilityTheory
+
+/-- Cumulative distribution function of a real measure. -/
```
Can you add a warning that it only really makes sense for probability measures, and add a pointer to `ofReal_cdf` (which is maybe the more usual definition of the cdf)?

sgouezel reviewed (commented):
It's a little bit surprising at first to see the cdf (which is a basic object) defined in terms of much more complicated objects, but in the end I agree that the API is pretty convenient. Maybe add a comment about that in the file-level docstring?

RemyDegenne commented:
I added comments about the definition in the file docstring and in the docstring of the definition. I also added a new lemma `cdf_eq_toReal` to complement `ofReal_cdf`. Finally, I made the measure argument explicit in all lemmas and removed two `@`.
